README.md

```diff
@@ -0,0 +1,116 @@
+  <strong>
+    <h1 align="center" >create-express-ts-api</h3>
+<p align="center">
+    <strong>
+      A starter to kit for developing  REST API in typescript and express.
+  </p>
+<!-- PROJECT SHIELDS -->
+<br></br>
+
+![restapi](https://user-images.githubusercontent.com/72249059/197589566-b2de5f90-1ddb-4d89-9811-e4da022bdd54.jpg)
+
+
+<br />
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#getting-started">Getting Started </a>
+    </li>
+    <li>
+      <a href="#Tech-Stack-Used">Tech stack used</a>
+    </li>
+    <li><a href="#Understanding-the-folder-structure">Understanding the folder structure</a></li>
+    <li><a href="#Setting-up-Development-DB">Setting up Development DB</a></li>
+    <li><a href="#Documentation">Documentation</a></li>
+  </ol>
+</details>
+
+---
+## Getting Started
+
+Follow the instructions to set up the project on your local machine.
+
+### Prerequisites
+
+Install [NodeJS LTS](https://nodejs.org/en/)
+
+- npm
+
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Initializing the project
+
+1. Initiate your project by using the following npm command:
+
+   ```sh
+   npx create-express-ts-api  <project-name>
+   ```
+
+   example : `npx create-express-ts-api Firstapi`
+
+3. Install NPM packages
```

Comment:
correct the sequence pls, it should be 2 after 1

Reply:
done